types/react-native-maps/index.d.ts

```diff
@@ -0,0 +1,522 @@
+//                 Naoufal El Yousfi <https://github.com/nelyousfi>
+//                 Alex Dunne <https://github.com/alexdunne>
+//                 Manuel Alabor <https://github.com/swissmanu>
+//                 Michele Bombardi <https://github.com/bm-software>
+//                 Tanguy Krotoff <https://github.com/tkrotoff>
+//                 Alexander T. <https://github.com/a-tarasyuk>
+//                 Martin van Dam <https://github.com/mvdam>
+//                 Kacper Wiszczuk <https://github.com/esemesek>
+//                 Ryan Nickel <https://github.com/mrnickel>
+//                 Souvik Ghosh <https://github.com/souvik-ghosh>
+//                 Cheng Gibson <https://github.com/nossbigg>
+//                 Saransh Kataria <https://github.com/saranshkataria>
+//                 Francesco Moro <https://github.com/franzmoro>
+//                 Wojciech Tyczynski <https://github.com/tykus160>
+//                 Paito Anderson <https://github.com/PaitoAnderson>
+//                 Peter Zavadsky <https://github.com/zavadpe>
+//                 Hein Rutjes <https://github.com/IjzerenHein>
+//                 Dan Tamas <https://github.com/rborn>
+//                 Alberto Martínez <https://github.com/aMarCruz>
+//                 Yusuke Shibahara <https://github.com/unosk>
+//                 Denis Oblogin <https://github.com/ukrbublik>
+//                 Tim O. Peters <https://github.com/timotew>
+//                 P.J. Swesey <https://github.com/swese44>
+//                 MarkRunWu <https://github.com/MarkRunWu>
+//                 RalfNieuwenhuizen <https://github.com/RalfNieuwenhuizen>
+//                 Rafael Lincoln <https://github.com/rafaellincoln>
+//                 Radovan Šmitala <https://github.com/radeno>
+//                 Ivan Pusic <https://github.com/ivpusic>
+//                 Michael Elsdörfer <https://github.com/miracle2k>
+//                 Mellet Solbakk <https://github.com/mellet>
+//                 Mellet Solbakk <https://github.com/mellet>
+//                 Nizam Moidu <https://github.com/netmaxt3r>
+//                 Marcel Kalveram <https://github.com/marcelkalveram>
+//                 Jason Li <https://github.com/lijason103>
+//                 JulienR2 <https://github.com/julienR2>
+//                 Ho Yin (Michael) Cheng <https://github.com/hysan>
+//                 Guilherme Eduardo Gulinoski Teixeira <https://github.com/GuilhermeEd>
+//                 lantz moore <https://github.com/mizerlou>
+//                 Almir Kadric <https://github.com/AlmirKadric>
+//                 origins-hongyi <https://github.com/fozhao>
+//                 Christoph <https://github.com/Stophface>
+//                 Budi Adiono <https://github.com/budiadiono>
+//                 Alexander Veligurov <https://github.com/alvelig>
+//                 Adrian Stutz <https://github.com/sttz>
+//                 Chris Shaffer <https://github.com/methodbox>
```

Comment:
Why is this list so much longer than the one of RN? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0d753f7c4c38f158613c7b0d22a554900a09afc7/types/react-native/index.d.ts#L3-L23

Reply:
Because the last one is me ;) and the rest are from the actual types contributors from `react-native-maps` types - their `index.d.ts`.

I’ve proposed to them already moving this to @types; before they process my PR with them to remove their `index.d.ts` we are waiting for this to be completed.

However, I included them because those are ultimately the people who contributed the types, with the exception of one that I corrected.

Reply:
Totally, but do they realise that this would mean they will start receiving notifications for each issue/PR about these types on this repo?

Reply:
Oh! I did not realize that, either. I can remove that then if that’s the best move?

Reply:
Well, I've removed them. I guess they can be added later if necessary.

Reply:
I always struggle with this header. The name makes it seem like it’s just an acknowledgement of who contributed, not that it has the side-effect of signing up as maintainer 🤷‍♂ So yeah, I would err on the side of caution too. Perhaps let them know on the upstream PR that they can sign-up as maintainer here.